Makefile

```diff
@@ -8,7 +8,7 @@ GOTOOLS = \
 
 INCLUDE = -I=. -I=${GOPATH}/src -I=${GOPATH}/src/github.com/gogo/protobuf/protobuf
 
-all: install test
+all: protoc install test metalinter
```

Comment:
`metalinter` - no such target! rename to `metalinter_test`

Reply:
lets rename metalinter_test to metalinter?

Reply:
@zramsay made it a standard across all the repos. this might screw some things in the future.

Reply:
k thank you.  the order here was important because you need to protoc before you can compile/install, and it's nice to compile before you test.  (actually we should probably build before we test, and install after the tests pass, because install adds something to your GOPATH/bin. Then metalinter can come last because it's least important, and it gets in the way of development flow.

Reply:
that just seems like overkill.  can we just use one? https://github.com/golang/lint ?

Reply:
it's strange naming.  just call it `make metalinter` and `make metalinter_all`.
making it a goal to use all of metalinter's linters seems like overkill. I don't think that should be a goal.